atomicapp/nulecule/main.py

```diff
@@ -54,7 +53,8 @@ class NuleculeManager(object):
     """
 
     def __init__(self, app_spec, destination=None,
-                 cli_answers=None, answers_file=None):
+                 cli_answers=None, answers_file=None,
+                 answers_format=ANSWERS_FILE_SAMPLE_FORMAT):
```

Comment:
should probably be `answers_format=None` since we set this in the first line of the function anyway